server/src/main/java/org/apache/druid/server/emitter/EmitterModule.java

```diff
@@ -130,10 +131,10 @@ public void inject(Injector injector)
     {
       final List<Binding<Emitter>> emitterBindings = injector.findBindingsByType(new TypeLiteral<Emitter>(){});
 
-      emitter = findEmitter(emitterType, emitterBindings);
-
-      if (emitter == null) {
+      if (Strings.isNullOrEmpty(emitterType)) {
```

Comment:
I'm not sure defaulting to noop if the emitter string is empty is a good idea either.

Reply:
I think this is fine as this is the default

Reply:
Oops, I forgot to reply here. I did it this way because we _do_ want to default to noop if it's null (not specified), and I was unsure how good the null vs empty distinction is preserved throughout the entire property handling system. (Like, I didn't want some unexpected behavior to happen if, somewhere, a null got translated to an empty string or vice versa.)

Reply:
@gianm could you please add this reasoning to a comment?

Reply:
I added comments for the above points.